pom.xml

```diff
@@ -94,7 +92,6 @@
     <version.org.jboss.xnio>3.2.0.Final</version.org.jboss.xnio>
     <!-- overwrites the version of jroups in ip-bom because the needed artifacts don't exit it the ip-bom version -->
     <version.org.jgroups>3.2.13.Final</version.org.jgroups>
```

Comment:
jgroups override can be removed as well, CR29 contains 3.2.13.Final - https://github.com/jboss-integration/jboss-integration-platform-bom/blob/6.0.0.CR29/pom.xml#L300